theano/sandbox/cpu_corr/tests/test_corr.py

```diff
@@ -0,0 +1,242 @@
+from nose.plugins.skip import SkipTest
+from nose.plugins.attrib import attr
+import numpy
+
+import theano
+import theano.tensor as T
+from theano.tests import unittest_tools as utt
+from theano.sandbox import cpu_corr
+from theano.tensor.nnet import conv
+from theano.tensor.basic import _allclose
+
+
+class TestCorr2D(utt.InferShapeTester):
+    mode = None
```

Comment:
use:

mode = "FAST_RUN" if theano.config.mode = "FAST_COMPILE" else None

Or implement a perform method. I would go with the fastest change (changing mode).